mountetna.github.io/magma.md

```diff
@@ -322,8 +322,12 @@ The Magma Query API lets you pull data out of Magma through an expressive query
 
 - `project_name` - the project key
 - `query` - An array of query predicates (see below for details)
+- `format` - Optional. If `tsv` is provided, a TSV is returned instead of a JSON paylod. By default, a JSON payload is provided.
+  - `user_columns` - If `format=tsv`, can provide an array of string values that act as column labels. The length of this array must match the number of requested columns.
+  - `expand_matrices` - If `format=tsv`, boolean value to expand matrix attributes into individual columns, with one matrix data point per column.
+  - `transpose` - If `format=tsv`, boolean value to transpose the resulting TSV. This means that rows become columns, and vice versa.
```

Comment:
I wonder if the extra indentation here might be confusing to users about whether these inputs are provided the same way as other inputs?  Perhaps instead of indenting over and the text "If `format=tsv`", could keep same indentation and change text to "Ignored unless `format="tsv"`".  Or keep the extra indents but *explain the indent* via a title level "Additional inputs ignored unless `format="tsv"` + then the requisite extra indent level for these inputs.